### DISCUSSION TITLE
Question - Is it possible to split one album between multiple artists in Fields

### DISCUSSION THREAD

**Discussion by bukakow** [reupen/columns_ui#344] [General] [answered]
I have a collab album in which every song has the Artist tag **Artist1; Artist2**.

Is it possible for the Fields window to be available simultaneously as **Artist1** as well as **Artist2**, instead of a new thing **Artist1, Artist 2**?

**Comment by reupen** [comment]
Hello,

You can use the new Discussions tab for questions now ;)

I'm not sure which panel you're referring to (I'm not aware of one named Fields). Could you confirm the name of the panel or post a screenshot?

**Comment by bukakow** [comment]
Preferences -> Columns UI -> Filters -> Fields

![obraz](https://user-images.githubusercontent.com/36563739/94453802-65947780-01b1-11eb-8ae8-c934cfeb8eb3.png)

**Comment by reupen** [comment, marked as answer]
If you enter only the field name (without % symbols) in preferences, then it will show each value as a separate item.

It's not possible to use title formatting and keep the values as separate items.
